js/popover.js

```diff
@@ -65,6 +65,10 @@
     var $e = this.$element
     var o  = this.options
 
+    if (!/string|function/.test(typeof o.content)) {
+      throw new Error('Popover received a(n) ' + typeof o.content + '. Expected a string or a function.')
```

Comment:
Fixed @hnrch02. Thanks for the suggestion :smiley:. I agree that wording works better.